src/extensions/surveys.tsx

```diff
@@ -768,6 +777,7 @@ export function Questions({
         const nextStep = getNextSurveyStep(survey, displayQuestionIndex, res)
         if (nextStep === SurveyQuestionBranchingType.End) {
             sendSurveyEvent({ ...questionsResponses, [responseKey]: res }, survey, posthog)
+            onSurveyDismissedOrSent()
```

Comment:
curious why we call this function here but not `handleCloseSurveyPopup`?

Reply:
`handleCloseSurveyPopup` is for when pressing the X button (dismissing the survey)

sending the survey is another action than closing/dismissing it